exporter/sematextexporter/factory.go

```diff
@@ -46,11 +49,17 @@ func createDefaultConfig() component.Config {
 		},
 		QueueSettings: exporterhelper.NewDefaultQueueConfig(),
 		MetricsConfig: MetricsConfig{
+			MetricsEndpoint: usMetricsEndpoint,
```

Comment:
Why set default metrics endpoint? Currently there is no default and the endpoint is set based on region in `Config.Validate`. Any reason to change this?